flink-connector-debezium/pom.xml

```diff
@@ -47,9 +47,19 @@ under the License.
                     <groupId>org.glassfish.jersey.core</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>jackson-databind</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.13.3</version>
```

Comment:
The jackson version should be maintained as a constant in the pom.xml of the `flink-cdc-connectors`.